src/main/java/hudson/remoting/Engine.java

```diff
@@ -783,9 +783,9 @@ public void closeRead() throws IOException {
                         client.getProperties().put(ClientProperties.SSL_ENGINE_CONFIGURATOR, sslEngineConfigurator);
                     }
                 }
-                if (!succeedsWithRetries(this::pingSuccessful)) {
-                    return;
-                }
+//                if (!succeedsWithRetries(this::pingSuccessful)) {
+//                    return;
+//                }
```

Comment:
this was failing both the `-noCertificateCheck` as well as when passing the `-cert` with error,
(notice the path `/login`)
```
INFO: https://<hostname>:8443/login is not ready
javax.net.ssl.SSLHandshakeException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
```

Reply:
Looks like a regression in #771.